packages/components/tooltip/src/tooltip.tsx

```diff
@@ -62,41 +62,44 @@ const Tooltip = forwardRef<"div", TooltipProps>((props, ref) => {
   const {ref: tooltipRef, id, style, ...otherTooltipProps} = getTooltipProps();
 
   const animatedContent = (
-    <div ref={tooltipRef} id={id} style={style}>
-      <LazyMotion features={domAnimation}>
-        <m.div
-          animate="enter"
-          exit="exit"
-          initial="exit"
-          variants={TRANSITION_VARIANTS.scaleSpring}
-          {...mergeProps(motionProps, otherTooltipProps)}
-          style={{
-            ...getTransformOrigins(placement),
-          }}
-        >
-          <Component {...getTooltipContentProps()}>{content}</Component>
-        </m.div>
-      </LazyMotion>
+    <div key={id + "-tooltip-content"} ref={tooltipRef} id={id} style={style}>
+      <m.div
+        key={id + "-tooltip-inner"}
+        animate="enter"
+        exit="exit"
+        initial="exit"
+        variants={TRANSITION_VARIANTS.scaleSpring}
+        {...mergeProps(motionProps, otherTooltipProps)}
+        style={{
+          ...getTransformOrigins(placement),
+        }}
+      >
+        <Component {...getTooltipContentProps()}>{content}</Component>
+      </m.div>
     </div>
   );
 
   return (
-    <>
+    <LazyMotion features={domAnimation}>
       {trigger}
-      {disableAnimation && isOpen ? (
-        <OverlayContainer portalContainer={portalContainer}>
-          <div ref={tooltipRef} id={id} style={style} {...otherTooltipProps}>
-            <Component {...getTooltipContentProps()}>{content}</Component>
-          </div>
-        </OverlayContainer>
+      {disableAnimation ? (
+        isOpen && (
+          <OverlayContainer portalContainer={portalContainer}>
```

Comment:
do we need the key here?

Reply:
The motion docs recommend that any children of `AnimatePresence` always have a key, but I did some testing and the tooltip works properly with all keys removed, on the `OverlayContainer` and the keys inside `animatedContent`. So I could probably remove those as well.

I did some research and apparently the reason it works with no keys is because framer motion has internal fallback mechanisms (component type as key, internal instance keys, component order) for tracking presence when there are no keys, and it works well here since it is a single child with a predictable structure.

I could see the argument for the `OverlayContainer` key go either way, up to you.